mlflow/tracking/client.py

```diff
@@ -189,6 +350,31 @@ def delete_experiment(self, experiment_id):
         Delete an experiment from the backend store.
 
         :param experiment_id: The experiment ID returned from ``create_experiment``.
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            # Create an experiment name, which must be unique and case sensitive
+            client = MlflowClient()
+            experiment_id = client.create_experiment("New Experiment")
+            client.delete_experiment(experiment_id)
+
+            # Examine the deleted experiment details. Deleted experiments
+            # are moved to a .thrash folder under the artifact URI location top
```

Comment:
```suggestion
            # are moved to a .trash folder under the artifact URI location top
```

nit

Reply:
lol, nice catch! Will fix it in the fluent APIs too. :)